Block/Redirect/Redirect.php

```diff
@@ -474,11 +474,7 @@ public function getMd()
      */
     public function getTermUrl()
     {
-        if ($termUrl = $this->getPayment()->getAdditionalInformation('termUrl')) {
-            return $termUrl;
-        }
-
-        throw new AdyenException("No termUrl is provided.");
+        return $this->getUrl('adyen/transparent/redirect', ['_secure' => $this->_getRequest()->isSecure()]);
```

Comment:
This might fix #854 too.

Reply:
Hello @fahu, 
We created a separate ticket to investigate it